repository-data/application/src/main/resources/hcm-config/namespaces/hee/publicationLandingPage.yaml

```diff
@@ -278,38 +262,30 @@ definitions:
             /cluster.options:
               jcr:primaryType: frontend:pluginconfig
               nodetypes: ['hee:logoGroup']
-          /publicationProfessions:
+          /classifiable:
```

Comment:
Unfortunately not. Since this is the default taxonomy field and always has to have this name. The rule seemed to be that you could have as many taxonomy fields on a document type but the first one always had to be implemented using the Essentials component and part of that is the name of the field

Reply:
Ah Ok 👌